modules/social_features/social_group/modules/social_group_invite/src/Plugin/Block/SocialInviteLocalActionsBlock.php

```diff
@@ -154,4 +153,11 @@ public function build() {
     return $build;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheMaxAge() {
```

Comment:
This might not permanently help as this block is rendered here https://github.com/goalgorilla/open_social/blob/10.0.x/modules/social_features/social_group/modules/social_group_invite/social_group_invite.module#L122